src/applications/disability-benefits/686/config/form.js

```diff
@@ -398,10 +571,15 @@ const formConfig = {
                 dateOfMarriage: currentOrPastDateUI(
                   'When did you get married?',
                 ),
-                locationOfMarriage: {
-                  'ui:title':
-                    'Where did you get married? (city and state or foreign country)',
-                },
+                locationOfMarriage: _.merge(
+                  createLocationUISchemaForKey(
+                    'marriages[INDEX].locationOfMarriage',
+                  ),
+                  {
+                    'ui:title':
+                      'Where did you get married? (city and state or foreign country)',
```

Comment:
Would it make sense to add a title parameter to `createLocationUISchemaForKey` to avoid this merge?